tools/osbuilder/README.md

```diff
@@ -95,6 +95,9 @@ filesystem components to generate an initrd.
 3. When generating an image, the initrd is extracted to obtain the base rootfs for
 the image.
 
+The default distro used to build a rootfs is CentOS, to use a different one, you can set `DISTRO=<your_distro>`.
+For example `make USE_DOCKER=true DISTRO=ubuntu rootfs` will make Ubuntu rootfs using Docker.
```

Comment:
How about `CentOS is the default distro for building the rootfs to use ...`

Reply:
or just `used to` -> `is used to`

Reply:
I liked that suggestion. Other than that, looks good to me.